bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -103,331 +97,68 @@ spec:
             memory: 300Mi
         terminationMessagePolicy: FallbackToLogsOnError
 
-      # nbdb: the northbound, or logical network object DB. In raft mode 
-      - name: nbdb
-        image: "{{.OvnImage}}"
+      - name: sb-ovsdb
+        image: "quay.io/mmahmoud/ovsdb-etcd:latest"
         command:
         - /bin/bash
         - -c
         - |
-          set -xe
-          if [[ -f /env/_master ]]; then
-            set -o allexport
-            source /env/_master
-            set +o allexport
-          fi
-
-          bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
-          # initialize variables
-          ovn_kubernetes_namespace=openshift-ovn-kubernetes
-          ovndb_ctl_ssl_opts="-p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt"
-          transport="ssl"
```

Comment:
we have not added SSL support to ovsdb-etcd.
Therefore we have 2 options:
add SSL support to ovsdb-etcd or to configure all ovsdb clients to work over plain TCP protocol.